packages/hts_shrink/hts_shrink.2.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/hts_shrink"
+bug-reports: "https://github.com/UnixJunkie/hts_shrink/issues"
+dev-repo: "git+https://github.com/UnixJunkie/hts_shrink.git"
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune"
+  "batteries"
+  "dolog"
+  "parmap"
+  "parany"
+  "minicli"
```

Comment:
I believe these dependencies need a lower-bound check as they became incompatible somewhat recently (e.g. since 5.0.0 for `minicli`)

Reply:
Probably, but unless there is a tool that allows to discover those bounds automatically,
I am not going to look for them.